### ISSUE TITLE
json source data instead of path

### ISSUE THREAD

**Issue by valentinbdv** [chenqingspring/ng-lottie#17]
Hi,

I am not sur if this is possible but let me explain my problem : We have more and more lottie animations in our application which make a lot of http requests and slow down the loading time. 

I was wondering if we could put the lottie data.json as a source instead of a path. Thus we could merge all our lottie's json files to have only one http request then split it to distribute it correctly in each animation.

Thanks!

**Comment by hamdiwanis** [comment]
Hello, you can use animationData property on the option object instead of path :)

**Comment by valentinbdv** [comment]
Yes, actually I test it yesterday and it works just fine! Thanks

**Comment by ElAndy94** [comment]
Thank you, this also solved my problem.
